addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/ContentSecurityPolicyScanRule.java

```diff
@@ -121,10 +121,11 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
         if (!xcspOptions.isEmpty()) {
             raiseAlert(
                     Constant.messages.getString(MESSAGE_PREFIX + "xcsp.name"),
-                    Constant.messages.getString(MESSAGE_PREFIX + "xcsp.desc"),
+                    Constant.messages.getString(MESSAGE_PREFIX + "xcsp.otherinfo"),
                     getHeaderField(msg, HTTP_HEADER_XCSP).get(0),
                     cspHeaderFound ? Alert.RISK_INFO : Alert.RISK_LOW,
-                    xcspOptions.get(0));
+                    xcspOptions.get(0),
+                    PLUGIN_ID + "-1");
```

Comment:
btw, it could just pass the postfix and have the concatenation in the `raiseAlert`.